backend/backend.go

```diff
@@ -6,10 +6,11 @@ import (
 
 	"github.com/opsfactory/kappa/backend/docker"
 	"github.com/opsfactory/kappa/config"
+	kappaevent "github.com/opsfactory/kappa/container/event"
 )
 
 type Backend interface {
-	Monitor(events <-chan string)
+	Monitor(events chan<- *kappaevent.Event)
```

Comment:
@atosatto  Passing a pointer to a chan usually is not a good idea. Also because here doesn't seem that there is a need to do so. 
The underlying functions should not be able to modify a kappa event IMHO this could also break the whole thing.

https://play.golang.org/p/VpXWAxotuR